.buildkite/hooks/pre-exit

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+echo ${OSTYPE}
```

Comment:
this is still being tested, might change depending on what works best on buildkite